lambda_src/drivers/destination_s3.py

```diff
@@ -89,6 +87,7 @@ def initialize(destination: Text, batch_id: Text):
     # Regex captures characters after and including the rightmost '/' in a path,
     # which are then replaced with a '/', e.g. '/a/b/c' -> '/a/b/'
     prefix_folder = re.sub(r'/[^/]*$', '/', prefix) if '/' in prefix else ''
+    prefix_folder = re.sub(r'/{\s*sha2\s*}.*', '/', prefix_folder, flags=re.IGNORECASE)
```

Comment:
this seems a little bit too specific -- could you change it to something that doesn't explicitly say "sha2" in it? if you'd prefer to keep it, let's make it a global variable so that it's defined once and used twice making it impossible for the two to be out of sync.

Reply:
maybe `r'/{[^}]}.*`'"?

Reply:
> this seems a little bit too specific -- could you change it to something that doesn't explicitly say "sha2" in it? if you'd prefer to keep it, let's make it a global variable so that it's defined once and used twice making it impossible for the two to be out of sync.

Will do, I'd kept the name similar to Snowflake's [SHA](https://docs.snowflake.com/en/sql-reference/functions/sha2#syntax) function. Maybe something more generic such as `{hash}`?

Reply:
> maybe `r'/{[^}]}.*'`?

Did you mean `r'/{[^}].*`?

I suppose there wouldn't be cases where a `{foobar}` in the path is intended to be anything besides a parameter, so should be fine?

Reply:
Had pushed a [global variable](https://github.com/Snowflake-Labs/geff/pull/64/commits/0c0773309bc50ed6ede340e481f8a181fe509a6f) for the hash param since we don't support any other brace'd paramaters, but keeping it more generic seems to make more sense now.